app/controllers/workouts_controller.rb

```diff
@@ -13,8 +13,7 @@ def new # GET /workouts/new - just presents the form to the user
   end
 
   def create  # POST /workouts - processes the form
-    @workout = Workout.new(workout_params)
-    @workout.user_id = current_user.id
+    @workout = current_user.workouts.build(workout_params)
```

Comment:
try to keep associations implied by chaining the constructor onto the collection (user.workouts.build)